src/esw/brushless.py

```diff
@@ -4,45 +4,87 @@
 import asyncio
 import rospy
 import math
+from abc import ABC, abstractmethod
 from geometry_msgs.msg import Twist
-from typing import NoReturn
+from typing import List, Dict
 from sensor_msgs.msg import JointState
-from mrover.msg import DriveStatus, MoteusState as MoteusStateMsg
+from mrover.msg import MoteusState as MoteusStateMsg, MotorsStatus
 import moteus
 import moteus.multiplex as mp
 import io
 
 
 class CommandData:
-    DEFAULT_TORQUE = 0.3
-    MAX_TORQUE = 0.5
+    MAX_TORQUE = 0.3
     POSITION_FOR_VELOCITY_CONTROL = math.nan
-    VELOCITY_LIMIT_REV_S = 50  # DO NOT CHANGE THIS HAPHAZARDLY. DERIVED FROM TESTING.
+    VELOCITY_LIMIT_REV_S = 80  # DO NOT CHANGE THIS HAPHAZARDLY. DERIVED FROM TESTING.
     ZERO_VELOCITY = 0.0
 
     def __init__(
         self,
         position: float = POSITION_FOR_VELOCITY_CONTROL,
         velocity: float = ZERO_VELOCITY,
-        torque: float = DEFAULT_TORQUE,
+        torque: float = MAX_TORQUE,
     ):
         self.position = position
         self.velocity = max(-CommandData.VELOCITY_LIMIT_REV_S, min(CommandData.VELOCITY_LIMIT_REV_S, velocity))
         self.torque = max(0, min(CommandData.MAX_TORQUE, torque))
 
 
 class MoteusData:
-    def __init__(self, position: float, velocity: float, torque: float):
+    def __init__(
+        self,
+        position: float = CommandData.POSITION_FOR_VELOCITY_CONTROL,
+        velocity: float = CommandData.ZERO_VELOCITY,
+        torque: float = CommandData.MAX_TORQUE,
+    ):
         self.position = position
         self.velocity = velocity
         self.torque = torque
 
 
 class MoteusState:
+    # TODO: Make MoteusState more object-oriented. Currently it's mostly acting as a namespace.
```

Comment:
Is this done?

Reply:
I think we should make it a github issue and have a younger member pick it up, could remove the todo from the code.